ci/build.sh

```diff
@@ -0,0 +1,62 @@
+set -e
+
+cd $1
+
+PROJECT_DIR=`pwd`
+
+mkdir build
+cd build
+
+case ${BUILD_TARGERT} in
+	windows_msvc)
+		if [ "${Platform}" = "x64" ]; then
+			PRJ_GEN="${PRJ_GEN} Win64"
+		fi
+	;;
+	windows_mingw)
+		if [ "${Platform}" = "x64" ]; then
+			MINGW_PATH=/C/mingw-w64/x86_64-6.3.0-posix-seh-rt_v5-rev1/mingw64
+			export RABBIT_BUILD_CROSS_HOST=x86_64-w64-mingw32
+			export RABBIT_BUILD_CROSS_SYSROOT=${MINGW_PATH}/${RABBIT_BUILD_CROSS_HOST}
+		else
+			MINGW_PATH=/C/mingw-w64/i686-6.3.0-posix-dwarf-rt_v5-rev1/mingw32
+			export RABBIT_BUILD_CROSS_HOST=i686-w64-mingw32
+			export RABBIT_BUILD_CROSS_SYSROOT=${MINGW_PATH}/${RABBIT_BUILD_CROSS_HOST}
+		fi
+		export PATH=${MINGW_PATH}/bin:$PATH
+		CMAKE_PARA="${CMAKE_PARA} -DCMAKE_TOOLCHAIN_FILE=$PROJECT_DIR/ci/CMake/Platforms/toolchain-mingw.cmake"
+
+		CONFIG_PARA="${CONFIG_PARA} --host=$RABBIT_BUILD_CROSS_HOST --target=$RABBIT_BUILD_CROSS_HOST"
+		if [ "$SHARED" = "OFF" ]; then
+			CONFIG_PARA="${CONFIG_PARA} --enable-static --disable-shared"
+		else
+			CONFIG_PARA="${CONFIG_PARA} --disable-static --enable-shared"
+		fi
+
+		cd ..
+		bash buildconf
+		cd build
+		../configure ${CONFIG_PARA} \
+			CFLAGS="--sysroot=${RABBIT_BUILD_CROSS_SYSROOT}" \
+			LDFLAGS="--sysroot=${RABBIT_BUILD_CROSS_SYSROOT}"
+		make -j`cat /proc/cpuinfo |grep 'cpu cores' |wc -l`
+		rm -fr *
+	;;
+esac
+
+cmake .. \
+	-G"${PRJ_GEN}" \
+	-DCMAKE_USE_OPENSSL=${OPENSSL} \
+	-DCMAKE_USE_WINSSL=${WINSSL}% \
+	-DBUILD_SHARED_LIBS=${SHARED} \
+	-DBUILD_TESTING=${TESTING} \
+	-DCURL_WERROR=ON \
+	-DENABLE_DEBUG=${ENABLE_DEBUG} \
+	-DCMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE="" \
+	-DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG="" \
+	-DCMAKE_INSTALL_PREFIX="${PROJECT_DIR}/install" \
+	-DCMAKE_BUILD_TYPE=${Configuration} \
+	${CMAKE_PARA}
+cmake --build . --config ${Configuration} --target install --clean-first
+
+cd ${PROJECT_DIR}
```

Comment:
Please use spaces instead of tabs to stay in sync with the rest of the codebase. Also add an ending newline to remove the red warning sign above. A shebang is also missing, so it's unclear if the code is meant to be bash or POSIX-shell compliant.